repose-aggregator/src/docs/asciidoc/architecture/system-model.adoc

```diff
@@ -119,6 +119,55 @@ If set to localhost, any physical node will match and run the *Repose* node.
      The default destination endpoint will be responsible for handling all traffic that is not explicitly routed to a different destination endpoint.
      Remember that a target can be specified rather than an endpoint if you want to use round-robin routing to a service cluster.
 
+=== Boolean Filter Determination
```

Comment:
In this case, I wonder if the word `Boolean` doesn't unnecessarily complicate the meaning of this section. Where does the boolean part come in? Because criteria evaluates to a boolean? Isn't that implied by determination? Because criteria must be boolean? That isn't actually the case. Is `Boolean Filter` a filter that needs determination? Well, no.

Just my 2c, but I like leaving `Boolean` out of the header here, but still using it where appropriate in the callouts below.

Reply:
Filter Activation Evaluation? Filter Activation Determination?

Reply:
Updated.